Dockerfile

```diff
@@ -34,4 +36,4 @@ COPY requirements.txt ./
 RUN npm install && \
   pip install --upgrade setuptools && \
   pip install --upgrade pip && \
-  pip install -r requirements.txt
+  pip install -r requirements.txt
```

Comment:
Can you add the newline back to the end of the file? And if your editor does that automatically, can you turn that feature off for this project? :)